sdk/nodejs/examples/http-server/src/index.ts

```diff
@@ -0,0 +1,55 @@
+const { Command } = require("commander");
+const zrok = require("zrok")
+
+
+const program = new Command();
+
+program
+  .command('http-server')
+  .version("1.0.0")
+  .description("command to host an HTTP Server")
+  .action(async () => {
+
+    // Load the zrok env
+    let root = zrok.Load()
+
+    // Authenticate with the Ziti network
+    await zrok.init( root ).catch(( err: Error ) => { console.error(err); return process.exit(1) });
+
+    // Set this to a larger value (e.g. 10) to trace lower-level Ziti SDK activity
+    zrok.setLogLevel(0)
+
+    // Create the zrok private share that will represent this web server
+    console.log("Now creating your private zrok share...")
+    let shr = await zrok.CreateShare(root, new zrok.ShareRequest(zrok.TCP_TUNNEL_BACKEND_MODE, zrok.PRIVATE_SHARE_MODE, "http-server", ["private"]));
```

Comment:
This is actually incorrect in the golang example, also:

```
	shr, err := sdk.CreateShare(root, &sdk.ShareRequest{
		BackendMode: sdk.TcpTunnelBackendMode,
		ShareMode:   sdk.PublicShareMode,
		Frontends:   []string{"public"},
		Target:      "http-server",
	})
```

... should be a `proxy` backend.

Might want to correct the Python and golang versions while we're making changes.

Reply:
done